### PR TITLE
Update folder structure in results/

### PR DESCRIPTION
**Pull request by tmarcu**
The logs, srpms, and debuginfo should all be split into their own
directory, so it is less confusing about which RPMs can be used when
creating mixes. They are mainly for extra metadata, and should not be
left in the main RPM results location.

Signed-off-by: Tudor Marcu <tudor.marcu@intel.com>

### CODE REVIEW

**Comment by tmarcu** [comment]
https://github.com/clearlinux/autospec/pull/176 is the corresponding PR that must be released at the same time.

**Comment by tmarcu** [comment]
@bryteise @phmccarty @ahkok I can't add reviewers for some reason... so tagging you instead.

**Comment by bryteise** [comment]
Seems fine to me.

**Comment by ahkok** [comment]
Fine with me, I wonder if we should use the same folder structure as we do in official repos, though? e.g. 

https://download.clearlinux.org/releases/23270/clear/

File: Makefile.common [hunk 1]
```diff
@@ -91,18 +91,24 @@ prebuild-checks:
...
+	@perl $(TOPLVL)/projects/common/logcheck.pl results/logs/build.log
+	@perl $(TOPLVL)/projects/common/count.pl results/logs/build.log &> testresults
 	@$(MAKE) checkblacklist
 
 #help build-nocheck: Same as 'make build', but do not run the package's test suite.
 build-nocheck: configemail $(SPECFILE) upstream $(SRPMFILE)
 	$(MOCK) $(SRPMFILE) --nocheck --result=results/ --no-cleanup-after --uniqueext=$(PKG_NAME)
-	@perl $(TOPLVL)/projects/common/logcheck.pl results/build.log
-	@perl $(TOPLVL)/projects/common/count.pl results/build.log &> testresults
+	mkdir -p results/logs
```
Comment (phmccarty) [inline Makefile.common]: This should be `mkdir -p results/logs results/debuginfo`, like you have for the `build` target, or else the mv command below for moving the debuginfo rpm will fail.

**Review by phmccarty — CHANGES_REQUESTED** [review]

File: Makefile.common (hunk 1 above)
Comment (tmarcu) [reply Makefile.common]: woo ty!

**Comment by tmarcu** [comment]
updated push

**Review by phmccarty — COMMENTED** [review]
+1

I will await a status update for https://github.com/clearlinux/autospec/pull/176 before considering a merge.

**Review by phmccarty — APPROVED** [review]

**Comment by phmccarty** [comment]
Fixed in 6613644b9c50742b4750b27de1d473811a46fd19

**Pull request closed without merging**